build.gradle

```diff
@@ -557,6 +557,57 @@ subprojects {
         includeClasses.add("org.opensearch.test.rest.ClientYamlTestSuiteIT")
         includeClasses.add("org.opensearch.upgrade.DetectEsInstallationTaskTests")
         includeClasses.add("org.opensearch.cluster.MinimumClusterManagerNodesIT")
+        includeClasses.add("org.opensearch.indices.IndicesRequestCacheIT")
```

Comment:
Obviously the plan to not automatically retry new flaky tests has not prevented the introduction of more flakiness. Should we just give up on this idea and automatically retry all tests?

Reply:
Just removed the filter with `includeClasses.add` in my latest commit, let me run the gradle check few times on the PR.

Reply:
I have updated the list to remove the tests which I did not see fail from past 1 year and with new flaky tests.

Reply:
I honestly think it might be better to remove the filter, particularly if we're going to add to the list without a good reason why we can't fix the flakiness. I don't like increasing the max retries though. How about removing the filter and setting max retries to 1? Rare flakiness won't block the build because a retry will almost always succeed. More severe flakiness will continue to block the build though. @prudhvigodithi  What do you think?

Reply:
Agree with you on removing the list, to see the same benefits should we continue with 3 as maxRetries ?

Reply:
@prudhvigodithi Can you remove the filter and set max retries to 1 in this PR and then run gradle check a few times to see what the success rate is?

The reason I'd like to keep retries to a minimum is so that retries only fix the really rare flakiness, but failures still happen for tests with high likelihood of failures.

Reply:
I have re-tried few times and got successful green CI in a row. Seen an unstable which indicates a re-try.